### DISCUSSION TITLE
Compatibility check comelit hfx-700m

### DISCUSSION THREAD

**Discussion by Medic97** [PricelessToolkit/ESPBell-LITE#16] [General]
I am not sure how to tell if mine is compatible. The door button only works when the video button is triggered. 

I would really love the smaller one that can trigger in HA. 

![20240524_152817](https://github.com/PricelessToolkit/ESPBell-LITE/assets/23649478/ad27ad87-483b-4c34-90f9-bf74d335291d)

**Comment by PricelessToolkit** [comment]
Hi, 2 wires = Digital, unfortunately, it is not supported.
